src/components/molecules/bar-chart.tsx

```diff
@@ -75,7 +76,9 @@ export const TrackerBarChart: FC<TrackerBarChartProps> = ({
   secondaryColor = '#ACAFC4',
   backgroundColor = colors.white
 }) => {
-  const {t} = useTranslation();
+  const {t, i18n} = useTranslation();
+  const dateLocale = getDateLocaleOptions(i18n);
+  const wideMonthLocales = ['bn'];
```

Comment:
Bengali `MMM` short form months are really wide and break the layout.

The design is changing here soon anyway and the new design will have space for it to no longer need special treatment.